integrationtests/tests/fixtures/mod.rs

```diff
@@ -810,6 +810,8 @@ impl FederationTest {
                     )
                     .await
                     .expect("DB Error");
+                    dbtx.notify_key(&fedimint_server::db::AcceptedTransactionKey(out_point.txid))
```

Comment:
Is this the right place to notify? The transaction hasn't committed yet

Reply:
Yes, the above comments should explain it.